php82.Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM --platform=linux/amd64 php:8.2.0RC3-fpm-alpine
+
+RUN apk --update add \
+  wget \
+  curl \
+  build-base \
+  libmcrypt-dev \
+  libxml2-dev \
+  pcre-dev \
+  zlib-dev \
+  autoconf \
+  oniguruma-dev \
+  openssl \
+  openssl-dev \
+  freetype-dev \
+  libjpeg-turbo-dev \
+  jpeg-dev \
+  libpng-dev \
+  imagemagick-dev \
+  imagemagick \
+  postgresql-dev \
+  libzip-dev \
+  gettext-dev \
+  libxslt-dev \
+  libgcrypt-dev &&\
+  rm /var/cache/apk/*
+
+RUN pecl channel-update pecl.php.net && \
+    pecl install mcrypt redis-5.3.7 && \
+    rm -rf /tmp/pear
+
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+
+RUN chmod +x /usr/local/bin/install-php-extensions && \
+    install-php-extensions sockets
```

Comment:
This is where I've had to pull in `sockets` manually as the build fails when I attempt install it with the rest of the php extensions due to a missing `linux-headers` dependency.